sdk/aqueduct/integrations/connect_config.py

```diff
@@ -275,6 +309,15 @@ def _prepare_slack_config(config: SlackConfig) -> _SlackConfigWithStringField:
     )
 
 
+def _prepare_aws_config(config: AWSConfig) -> _AWSConfigWithSerializedConfig:
+    return _AWSConfigWithSerializedConfig(
+        access_key_id=config.access_key_id,
+        secret_access_key=config.secret_access_key,
+        region=config.region,
+        k8s_serialized=(None if config.k8s is None else config.k8s.json(exclude_none=True)),
```

Comment:
nit: I think `config.k8s or config.k8s.json(exclude_none=True)` does what you want in a more succinct fashion.

Reply:
hmm won't this just return a bool?

Reply:
No it'll return the value of the first expression, unless it's evaluated to False (aka None in this case), whereas it'll use the second expression. It should do the same thing as your inline if-else statement.

Reply:
@kenxu95 honestly I find my original version much easier to understand. :) I'll keep it unless you feel strongly.

Reply:
and I don't think I understand your explanation still...I don't want it to return the value of the first expression (`config.k8s`) right? And if it is evaluated to None, how can I then return the second expression?

Reply:
`config.k8s or config.k8s.json(exclude_none=True)`

config.k8s.json(exclude_none=True) is the second expression that is returned.

Reply:
fine with me either way, I just think I saw this pattern in our javascript code too